TDAT: public void test bp resume with wrong arg ( ) throws exception {  <NL> try {  <NL> this . bp manager . resume ( null ) ;  <NL> fail ( " resume report success for null bpxid " ) ;  <NL>  }  <NL> catch ( bp manager exception e ) {  <NL>  }  <NL> catch ( exception ex ) {  <NL> fail ( " wrong exception is throwed : " + ex ) ;  <NL>  }  <NL>  <NL> bp thread resume request vo request2 = new bp thread resume request vo ( ) ;  <NL> request2 . set threadx id ( common config . thread _ id ) ;  <NL> try {  <NL>  / * bp response vo result = * / this . bp manager . resume ( request2 ) ;  <NL> fail ( " resume report success for wrong threadid " ) ;  <NL>  }  <NL> catch ( bp manager exception e ) {  <NL>  }  <NL> catch ( exception ex ) {  <NL> fail ( " wrong exception is throwed : " + ex ) ;  <NL>  }  <NL>  }  COM: <s> test of the business process resume by resume request with legal argument </s>